modules/fuse-box-loader-api/fusebox.js

```diff
@@ -276,10 +276,12 @@ function $import(name, o) {
             v: ref.versions,
         });
     };
-    locals.require.main = {
-        filename: $isBrowser ? "./" : g["require"].main.filename,
-        paths: $isBrowser ? [] : g["require"].main.paths,
-    };
+    if ($isBrowser || !g["require"].main) {
```

Comment:
I'm not sure I follow your question. 
`g["require"]` only gets evaluated if `$isBrowser` is not truthy. Is that check not sufficient?